client/post-editor/style.scss

```diff
@@ -67,77 +26,59 @@
 }
 
 .post-editor__inner {
-	padding: 0 24px;
 	position: relative;
-	width: 1040px;
-
-	display: flex;
-	flex: 1;
-	flex-basis: auto;
+}
 
-	box-sizing: border-box;
+.is-section-post .email-verification-notice,
+.post-editor__content {
+	margin: 0;
 
-	@include breakpoint( "<1040px" ) {
-		max-width: 100%;
+	@include breakpoint( ">660px" ) {
+		margin-left: ( $sidebar-width-min + 1 );
 	}
 
-	@include breakpoint( "<660px" ) {
-		margin-top: 32px;
-		padding: 0;
+	@include breakpoint( ">960px" ) {
+		margin-left: ( $sidebar-width-max + 1 );
 	}
 }
 
 .post-editor__content {
-	flex: 1;
 	position: relative;
+	backface-visibility: hidden;
+	perspective: 1000;
 
-
-	@include breakpoint( "<660px" ) {
-		flex: none;
-		transform: translateX( 0 );
-		transition: all 0.15s cubic-bezier(0.770, 0.000, 0.175, 1.000);
-		width: 100%;
-
-		.focus-sidebar & {
-			transform: translateX( 100vw );
-		}
+	@include breakpoint( ">660px" ) {
+		backface-visibility: visible;
+		perspective: none;
 	}
 }
 
 .post-editor__sidebar {
-	flex: 0 0 269px;
-	max-width: 269px;
-	margin-right: 40px;
-	order: -1;
-
-	@include breakpoint( "<660px" ) {
-		box-sizing: border-box;
-		max-width: 100vw;
-		width: 100%;
-		flex: none;
-		margin: 0;
-		padding: 0 20%;
-		position: absolute;
-		transform: translateX( -100vw );
-		transition: all 0.15s cubic-bezier(0.770, 0.000, 0.175, 1.000);
+	@extend .sidebar;
+}
 
-		.focus-sidebar & {
-			transform: translateX( 0 );
-			position: inherit;
-		}
-	}
+.editor__header,
+.editor .mce-container-body,
+.editor .tinymce {
+	max-width: 100%;
+	margin: 0 auto;
 
-	@include breakpoint( "660px-960px" ) {
-		max-width: 220px;
-		margin-right: 24px;
+	@include breakpoint( ">660px" ) {
+		width: 700px;
```

Comment:
This is getting applied to the toolbar as well, which creates this gap:

![screen shot 2016-01-05 at 5 47 53 pm](https://cloud.githubusercontent.com/assets/5835847/12130977/84234aee-b3d4-11e5-955c-b6654e982d06.png)

Added 63dff24 that seems to fix it, but there could be a better way?